htdocs/ticket/class/ticket.class.php

```diff
@@ -1454,6 +1463,9 @@ public function markAsRead($user, $notrigger = 0)
 		$error = 0;
 
 		if ($this->statut != self::STATUS_CANCELED) { // no closed
+			if (empty($notrigger)) {
+				$this->oldcopy = dol_clone($this, 1);
```

Comment:
Do you need param 2 = 1 ?
We use it when we want a "clone" that keep some share data (pointers on dynamic object for example like $this->db remains the same for example). Here we want it to read simple properties by triggers, so 
`$this->oldcopy = dol_clone($this)`
looks better.